hard_task: "([]<> unloaded) && ([]<> loaded) && ([]! r4)"
soft_task: ""
